### DISCUSSION TITLE
Remove Links to Simplify Left Sidebar Navigation

### DISCUSSION THREAD

**Discussion by dylsell** [deso-protocol/dips#29] [Medium CIPs]
**Problem**

The left sidebar takes up lots of real-estate, has potential to confuse new users, and has several duplicative endpoints. Understanding that this is solely a UI treatment to improve the eventual sunset of bitclout.com. But, i'd argue that this is important enough for the next X months of the site to sew up. 

**Proposed Solution**

Remove the following tabs 

- Buy BitClout 
- Buy Creator Coins (list of top accounts isn't a common user journey and this list is accessible in many other places) 
- Send Creator Coins (duplicative link exists in wallet -> holdings -> send) 
- Send BitClout (duplicative link exists in wallet -> send) 

- Add "Buy $CLOUT" to Wallet "home" (pictured below) 

<img width="602" alt="Screen Shot 2021-07-09 at 4 46 02 PM" src="https://user-images.githubusercontent.com/33188041/125134404-54c84900-e0d5-11eb-8aea-10bcbef3ba76.png">
 

The end result of these changes is a much simpler nav with all transactions related activities consolidated under the wallet. 

<img width="431" alt="Screen Shot 2021-07-09 at 4 38 16 PM" src="https://user-images.githubusercontent.com/33188041/125133920-8bea2a80-e0d4-11eb-8927-d5a3ca9e062a.png">

**Comment by HPaulson** [comment]
+1 for this. These tabs are clunky and very unnecessary -- can just remove the components entirely.

**Reply by tijno** [reply]
kinda need it on mobile to get to help :) 

thats about it.

**Comment by 0xNordian** [comment]
"Less is more", and for onboarding purposes and new user experience, I think this is a relevant point.

Thank you!

**Comment by tijno** [comment]
120%

**Comment by AikonMishima** [comment]
Agreed!

The dynamics of selling and buying should indeed be incorporated in the wallet section, as there’s no need for these shortcuts.

I would, however, propose to keep the « Buy Bitclout » button on there.
Reasoning behind this is to stimulate the purchase of $CLOUT.

**Reply by lazynina** [reply]
Agreed that we should definitely keep the Buy $CLOUT in the left bar. It needs to be easy to access and hiding it in the wallet will decrease visibility.

**Comment by lazynina** [comment]
I am in favor of simplifying the left sidebar but we need to keep the Buy $CLOUT option there IMO.

@maebeam @diamondhands0 what are your thoughts here?

**Reply by maebeam** [reply]
Unfortunately these links are some of the only user education the product currently has. Until we educate new users about buying and sending CLOUT and creator coins I think we probably need to keep these even though they feel a bit clunky

**Reply by AikonMishima** [reply]
If we could find a way to introduce a facilitating feature for user education, per example in the form of an overlay (previously proposed by @N0rdian in the form of « walkme ») or an all encompassing page; would that be sufficient for you to go through with the retirement of the prementioned buttons?

**Comment by omicreativedev** [comment]
What if **Wallet** was a selection dropdown box so that the proposed changes could be implemented (elsewhere) and the sidebar compressed somewhat, but upon clicking; offers shortcuts in dropdown menu— AND proposing a welcome tutorial/tour for first time users? Then eventually phasing out dropdown box if/when it appears that the tour is effective? Also maybe adding a wallet help doc and link may reduce support workload. I'm sure there are non-coders that wouldn't mind making docs for actual user experience (i.e. How To Send Creator Coins, How To Buy Creator Coins, How To Update Profile, etc.) **Add new _page_ for Help** with links to Restart Tour, Docs, FAQs, WhitePaper, Discord, Social, Help, etc. and put support email or form on page instead of mailto: link.

Agreed that Buy $CLOUT should remain in sidebar no matter what.

**Reply by AikonMishima** [reply]
This would indeed create a best of both worlds scenario.

Also, repurposing the « help » button with a new page would be very helpful.
Shall we create a secondary discussion just for gathering proposals for community input, @teambrixmusic ?
